src/admin/content/helpers/reducer.ts

```diff
@@ -1,22 +1,25 @@
 import { createReducer } from '../../../shared/helpers';
+import { ContentBlockConfig } from '../../content-block/content-block.types';
 
-import {
-	ContentEditBlocksAction,
-	ContentEditBlocksActionType,
-	ContentEditBlocksState,
-} from '../content.types';
+import { ContentEditAction, ContentEditActionType, ContentEditState } from '../content.types';
 
-export const CONTENT_EDIT_BLOCKS_INITIAL_STATE = (): ContentEditBlocksState => ({
-	cbConfigs: [],
+export const CONTENT_EDIT_INITIAL_STATE = (
+	cbConfigs: ContentBlockConfig[] = []
+): ContentEditState => ({
+	cbConfigs,
 });
 
-export const contentEditBlocksReducer = (initialState: ContentEditBlocksState) =>
+export const contentEditReducer = (initialState: ContentEditState) =>
 	createReducer(initialState, {
```

Comment:
deze createReducer is overbodig denk ik. Je hoeft ook de initialState niet mee te geven aan de contentEditReducer functie.
Zie voorbeeld: https://reactjs.org/docs/hooks-reference.html#usereducer

Reply:
Ik heb die createReducer functie hier gebruikt omdat die by default uw state returned als de action type niet klopt, ook omdat ik dan geen switch statement moest schrijven 😄